### DISCUSSION TITLE
Almost all lines in GC dump are UNKNOWN

### DISCUSSION THREAD

**Discussion by MaximTkachenko** [microsoft/perfview#1747] [Q&A]
I got a dump of a dotnet application from production using `dotnet-monitor`. When I open `dump.gcdump` file in PerfView and check `Heap Stacks` almost all lines looks like this:

![1ERoc](https://user-images.githubusercontent.com/1560515/200866491-0aaf778c-4fe2-443c-80ed-146358fafdfe.png)

dotnet 6, PerfView 2.0.77

How to get proper names of classes?

**Comment by brianrob** [comment]
@noahfalk is there a known issue around this?  I sort of remember hearing something about this.

**Comment by noahfalk** [comment]
If there is a known issue it isn't one I recall, but perhaps @davmason remembers better?

**Comment by MaximTkachenko** [comment]
I tried to analyze the dump with cli and result is the same

![image](https://user-images.githubusercontent.com/1560515/201096941-149854ab-c45c-4f6e-8ce0-a8a61a49518a.png)

So I suppose the problem is in the dump itself.

**Comment by MaximTkachenko** [comment]
Ok, it looks like I found the root cause https://github.com/dotnet/diagnostics/issues/1318#issuecomment-654350059

**Reply by brianrob** [reply]
@MaximTkachenko, were you just missing some of the required memory pages?
